uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/toil-request-ctrl.spec.js

```diff
@@ -348,6 +350,38 @@
             });
           });
         });
+
+        describe('when TOIL Request does not expire', function () {
+          beforeEach(function () {
+            AbsenceType.canExpire.and.returnValue($q.resolve(false));
+            initTestController({
+              contactId: 202,
+              leaveRequest: $ctrl.request
+            });
+          });
+
+          it('should set requestCanExpire to false', function () {
+            expect($ctrl.requestCanExpire).toBe(false);
+          });
+
+          describe('when request date changes', function () {
+            beforeEach(function () {
+              spyOn(AbsenceType, 'calculateToilExpiryDate');
+              $ctrl.request.to_date = new Date();
+              $ctrl.calculateToilExpiryDate();
+              $rootScope.$digest();
+            });
+
+            it('should not calculate the expiry date field', function () {
+              expect(AbsenceType.calculateToilExpiryDate)
```

Comment:
Yes, we do not follow strict 80 chars rule. The key is to keep the code readable. In this case 1 line is OK.